### DISCUSSION TITLE
Build Script For New Users

### DISCUSSION THREAD

**Discussion by BenKurrek** [near/near-sdk-js#31] [Q&A] [answered]
I’ve got a working demoable game written using the new JS sdk but currently everything is in a cloned version of the near-sdk-js repo since that’s where the builder.sh script is found. For new users that are using this sdk, are they forced to clone the near-sdk-js repo and use the builder.sh script there? or is there another way for users to start programming in JS such as just needing the actual shell script without cloning the whole repo?

As a follow up, now that i’m finished with this prototype, I’d like to make it open source. If I wanted to push it to github, do I need to push it along with the entire forked near-sdk-js repo?

**Comment by BenKurrek** [comment]
Answer from @volovyk-s: 

We are planning to publish js-sdk to npm. Ticket is already on the board. Maybe you will need some script anyway, not sure.
For now it works only inside of the repo.

**Reply by think-in-universe** [reply]
It's already published: https://www.npmjs.com/package/near-sdk-js

**Comment by volovyks** [comment, marked as answer]
Now you need to `npm install near-sdk-js` and modify your `package.json` `build` command to be `near-sdk build`.
Alternatevily, use this [template](https://github.com/near/near-sdk-js-template-project).
